setup.py

```diff
@@ -109,7 +109,7 @@ def is_platform_mac():
 _pxifiles = ['algos_common_helper.pxi.in', 'algos_groupby_helper.pxi.in',
              'algos_join_helper.pxi.in', 'algos_take_helper.pxi.in',
              'hashtable_class_helper.pxi.in', 'hashtable_func_helper.pxi.in',
-             'sparse_op_helper.pxi.in']
+             'sparse_op_helper.pxi.in', 'joins_func_helper.pxi.in']
```

Comment:
we will prob need to split out join as a separate cython module (``_join.pyx``), rather than including in algos.pyx which is massive now. (pls create a new issue for this)